bot/exts/easter/conversationstarters.py

```diff
@@ -3,13 +3,23 @@
 import random
 from pathlib import Path
 
+from discord import Embed
 from discord.ext import commands
 
+from bot.utils.decorators import override_in_channel
+
+
 log = logging.getLogger(__name__)
 
+
 with open(Path("bot/resources/easter/starter.json"), "r", encoding="utf8") as f:
```

Comment:
I'll leave this up to you, but I feel like we should use the `pathlib.Path.open` method here instead. They both do the same thing though, so it's completely a stylistic choice. If you do decide to change it here, make sure to change it below as well to keep it consistent.
```suggestion
with Path("bot/resources/easter/starter.json").open("r", encoding="utf8") as f:
```